omr.rc.in

```diff
@@ -0,0 +1,29 @@
+
+#include <windows.h>
+#include <winver.h>
+
+VS_VERSION_INFO VERSIONINFO
+ FILEVERSION 2,9,5,19691
+ PRODUCTVERSION 2,9,5,19691
+ FILEFLAGSMASK 0x3fL
+ FILEFLAGS 0x0L
+ FILEOS VOS_NT_WINDOWS32
+ FILETYPE VFT_DLL
+ FILESUBTYPE 0x0L
+BEGIN
+	BLOCK "StringFileInfo"
+	BEGIN
+		BLOCK "040904b0"
+		BEGIN
+			VALUE "CompanyName", "International Business Machines Corporation\0"
```

Comment:
@ChengJin01 @charliegracie 
1) By not having this file in the glue directory, it forces consumers to maintain a patch, which is not ideal (i doubt this file will change often, so probably not *too* painful but it's not in line with our consumption story).
2) It definitely should not default to creating a library owned by IBM. IBM can build such libraries, but OMR should not force others to create such libraries. The default name and legalcopyright should be an empty string, I think, based on a quick google search.

Maybe create OMR_COMPANY_NAME and OMR_COMPANY_COPYRIGHT (initially empty strings) but can be configured by consumers?